v19.2/insert.md

```diff
@@ -67,68 +67,81 @@ key. Using `ON CONFLICT` is therefore more flexible.
 
 ## Examples
 
-All of the examples below assume you've already created a table `accounts`:
-
-{% include copy-clipboard.html %}
-~~~ sql
-> CREATE TABLE accounts(
-    id INT DEFAULT unique_rowid() PRIMARY KEY,
-    balance DECIMAL
-);
-~~~
+{% include {{page.version.version}}/sql/movr-statements.md %}
```

Comment:
Left comments on `movr-statements.md` already, but wanna reiterate question of whether we can revamp to add "logic" to the text so we don't need `movr-statements-no-data.md`.

Reply:
Sure thing - let's talk about this.